### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by tanyouliang95**
- Update installation instruction according to https://github.com/open-rmf/rmf/issues/27.
- Update to `ignition-edifice` https://github.com/open-rmf/rmf_demos/pull/24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +34,7 @@ Install all non-ROS dependencies of RMF packages,
 sudo apt update && sudo apt install \
   git cmake python3-vcstool curl \
   qt5-default \
-  python3-shapely python3-yaml python3-requests python3-flask-cors \
+  ignition-edifice \
   -y
 python3 -m pip install flask-socketio
```
Comment (Yadunund) [inline README.md:39]: Can we push `flash_socketio` to upstream `rosdep`? Assuming rosdep will allow us to install the pip version of the package. If I remember correctly, the debian version was a lower version that what is required.

**Review by Yadunund — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tanyouliang95) [reply README.md:39]: From the [PR](https://github.com/ros/rosdistro/pull/29086) to `rosdep`, I did include `python3-flask-socketio` and it is available now. Yes you are right, the issue here is related to the official "ubuntu 20.04" package is using `flask_socketio` version `4.X`, yet our application requires `5.X`. 

The good news here is, the upcoming Ubuntu 21.04 will be using `socketio 5.X`  https://packages.ubuntu.com/source/hirsute/flask-socketio

Comment (Yadunund) [reply README.md:39]: I see, so as of now including `<exec_depend>python3-flask-socketio</exec_depend>` in `package.xml`s will result in rosdep installing `4.X`?

Comment (tanyouliang95) [reply README.md:39]: Yes, thats right.

**Pull request merged**